Yoast/wordpress-seo
# Update context/copy on the extensions page

## Summary

This PR can be summarized in the following changelog entry:

* Changed references from `licenses` to subscriptions and sites.

## Test instructions

This PR can be tested by following these steps:

* Go to the Premium item in the menu
* Check if the `More information` link works next to Buy premium
* Switch to Dutch and see that all the items for Local SEO can be read
* Hover the Buy buttons on the extensions and see the pointer change to a `hand`
* On installed but not activated extensions the copy should read "Activate your site on My Yoast"
* On installed and activated extensions the copy should read "Manage your subscription on My Yoast"

Fixes https://github.com/Yoast/wordpress-seo-premium/issues/1305
